mizar/obj/endpoint.py

```diff
@@ -460,6 +454,30 @@ def update_network_policy_protocol_port_egress(self, port_table):
         self.rpc.update_network_policy_protocol_port_egress(
             self, cidr_networkpolicy_list)
 
+    def update_pod_label_policy(self, policy_table):
+        cidr_networkpolicy_list = []
+        for item in policy_table:
+            cidr_networkpolicy = LabelPolicy(
+                item["vni"], item["pod_label_value"], "", item["bit_value"])
+            cidr_networkpolicy_list.append(cidr_networkpolicy)
+        self.rpc.update_pod_label_policy(cidr_networkpolicy_list)
+
+    def update_namespace_label_policy(self, policy_table):
+        cidr_networkpolicy_list = []
+        for item in policy_table:
+            cidr_networkpolicy = LabelPolicy(
+                item["vni"], "", item["namespace_label_value"], item["bit_value"])
+            cidr_networkpolicy_list.append(cidr_networkpolicy)
+        self.rpc.update_namespace_label_policy(cidr_networkpolicy_list)
+
+    def update_pod_and_namespace_label_policy(self, policy_table):
+        cidr_networkpolicy_list = []
+        for item in policy_table:
+            cidr_networkpolicy = LabelPolicy(
+                item["vni"], item["pod_label_value"], item["namespace_label_value"], item["bit_value"])
```

Comment:
Same here. If item[pod-lable], item[ns-label] is empty, can this continue to to work? If yes, we can remove update_pod_label_policy & update_namespace_label_policy

Reply:
if there is a rule "pod and namespace", it means the pod condition should be "and" namespace condition. And we cannot simply differentiate with pod condition only, namespace condition only, pod condition "or" namespace condition. So basically merge them together either not work, or have to add more flags and make it not clean.